dali/pipeline/operators/resize/resize.cc

```diff
@@ -92,35 +90,17 @@ void Resize<CPUBackend>::RunImpl(SampleWorkspace *ws, const int idx) {
                  "Resize expects interleaved channel layout (NHWC)");
   }
 
-  auto in_view = view<const uint8_t, 3>(input);
-  auto &kdata = kernel_data_[thread_idx];
-  kdata.requirements = Kernel::GetRequirements(
-      kdata.context,
-      in_view,
-      resample_params_[thread_idx]);
-  kdata.scratch_alloc.Reserve(kdata.requirements.scratch_sizes);
-  auto scratchpad = kdata.scratch_alloc.GetScratchpad();
-  kdata.context.scratchpad = &scratchpad;
-
-  auto &output = ws->Output<CPUBackend>(outputs_per_idx_ * idx);
-  const auto &input_shape = input.shape();
-
-  auto out_shape = kdata.requirements.output_shapes[0][0];
-  out_shape_[thread_idx] = out_shape.shape;
-
-  // Resize the output & run
-  output.Resize(out_shape_[thread_idx]);
-  auto out_view = view<uint8_t, 3>(output);
-  Kernel::Run(kdata.context, out_view, in_view, resample_params_[thread_idx]);
+  RunCPU(output, input, thread_idx);
 
   if (save_attrs_) {
-      auto &attr_output = ws->Output<CPUBackend>(outputs_per_idx_ * idx + 1);
+    auto &attr_output = ws->Output<CPUBackend>(outputs_per_idx_ * idx + 1);
+    auto &in_shape = input.shape();
 
-      attr_output.Resize(Dims{2});
-      int *t = attr_output.mutable_data<int>();
-      t[0] = out_shape[0];
```

Comment:
I think this was a bug. GPU variant uses input size, which seems more reasonable.